apps/examples/src/e2e/TestingView.tsx

```diff
@@ -0,0 +1,157 @@
+import React, {
+  Component,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import {Platform, Text, View} from 'react-native';
+import * as RNSVG from 'react-native-svg';
+import ViewShot from 'react-native-view-shot';
+
+const address = ['ios', 'web'].includes(Platform.OS) ? 'localhost' : '10.0.2.2';
+const wsUri = `ws://${address}:7123`;
+
+const TestingView = () => {
+  const wrapperRef = useRef<ViewShot>(null);
+  const [wsClient, setWsClient] = useState<WebSocket | null>(null);
+  const [renderedContent, setRenderedContent] = useState();
```

Comment:
> Since this function returns `React.createElement` we could at least type it as `ReactElement`
https://github.com/software-mansion/react-native-svg/pull/2338/files#r1698379189
```suggestion
  const [renderedContent, setRenderedContent] =
    useState<React.ReactElement | null>();
```

Reply:
Good point, thanks